Homepage work:

dwang19 opened the pull request:
- homepage.php view is now structured to house the dashboard
- styles.css is updated to format homepage as per homepage-wireframes
- a {players} stub is included in the "Players" panel to iterate 
  player information into the panel (this will be completed at a later 
  time)
- sub controllers created with code stubs
  Signed-off-by:David Wang lijen_dwang@hotmail.com

dwang19 committed:
Homepage work:

- homepage.php view is now structured to house the dashboard
- styles.css is updated to format homepage as per homepage-wireframes
- a {players} stub is included in the "Players" panel to iterate 
  player information into the panel (this will be completed at a later 
  time)
Signed-off-by:David Wang <lijen_dwang@hotmail.com>

nnanceraa commented:
- empty out the config/database.php, shouldnt have your set up in it
- delete or clear out the Welcome controller
- extra space in front of the load parser library

dwang19 committed:
Homepage:

My last pull request was rejected due to some minor corrections that needed to be done. This is the revised edition of my last pull request.
Signed-off-by:David Wang <lijen_dwang@hotmail.com>

dwang19 commented:
@nnanceraa corrections you requested have been applied, please review for merge.
